tests/topotests/bgp_path_selection/r1/bgpd.conf

```diff
@@ -0,0 +1,28 @@
+router bgp 65001
+ no bgp ebgp-requires-policy
+ neighbor 2.2.2.2 remote-as 65002
```

Comment:
Please use IPv4 documentation range. Ref: https://github.com/FRRouting/frr/blob/master/doc/developer/topotests.rst#writing-tests (^f + `Always use IPv4`).

Reply:
`2.2.2.2` is the loopback of the `r2` router

Reply:
Yeah, I meant to replace all Cloudflare and others IP ranges to documentation ranges :)

Reply:
I understand that for the documentation but for the topotest, the is no risk of a leak. We are not going to send any traffic to Cloudflare

Reply:
The idea isn't to send/not-send the traffic but to avoid abusing the ranges. There is even a talk from Cloudflare guys that complained when companies are using their ranges for everything.

Reply:
I have changed that